types/occ/types.go

```diff
@@ -0,0 +1,22 @@
+package occ
+
+import (
+	"errors"
+)
+
+var (
+	ErrReadEstimate = errors.New("multiversion store value contains estimate, cannot read, aborting")
+)
+
+// Abort contains the information for a transaction's conflict
+type Abort struct {
```

Comment:
i'm ok moving this to scheduler too if we want, otherwise if theres a concern of import cycle we can keep in types so it can be used by both `store` and `scheduler`